compose/cli/command.py

```diff
@@ -120,14 +118,14 @@ def unicode_paths(paths):
 
 def get_project(project_dir, config_path=None, project_name=None, verbose=False,
                 context=None, environment=None, override_dir=None,
-                compatibility=False, interpolate=True, environment_file=None):
+                interpolate=True, environment_file=None):
```

Comment:
If any part of the code using this function in a positional way. 
This would break that.

Reply:
Haven't found any in the compose code.

Reply:
:+1: